File: dataset/output/MISTRAL_codestral-latest/Task27_MISTRAL_codestral-latest.cpp
#include <vector>
int find_outlier(std::vector<int> integers) {
    int evens = 0, odds = 0, even = 0, odd = 0;
    for (int i = 0; i < integers.size(); i++) {
        if (integers[i] % 2 == 0) {
            evens++;
            even = integers[i];
        } else {
            odds++;
            odd = integers[i];
        }
    }
    return evens < odds ? even : odd;
}

File: dataset/output/MISTRAL_codestral-latest/Task27_MISTRAL_codestral-latest.java
public class Main {
    public static int find(int[] integers){
        int evenCount = 0, oddCount = 0;
        int even = 0, odd = 0;
        for(int i = 0; i < integers.length; i++){
            if(integers[i] % 2 == 0){
                evenCount++;
                even = integers[i];
            } else {
                oddCount++;
                odd = integers[i];
            }
        }
        return evenCount < oddCount ? even : odd;
    }
}